nerfstudio/models/splatfacto.py

```diff
@@ -657,8 +662,9 @@ def get_outputs(self, camera: Cameras) -> Dict[str, Union[torch.Tensor, List]]:
         # get the background color
         if self.training:
             optimized_camera_to_world = self.camera_optimizer.apply_to_camera(camera)[0, ...]
-
-            if self.config.background_color == "random":
+            if self.config.background_color == "auto":
+                background = torch.sigmoid(self.learnable_bg).to(self.device)
```

Comment:
Do we need `to(self.device)` here? I would assume self.learnable_bg will be converted to `self.device` automatically as part of `model.to(device)` in __init__ call of VanillaPipeline.

Reply:
testing on this